banyand/stream/stream_query_test.go

```diff
@@ -0,0 +1,532 @@
+// Licensed to Apache Software Foundation (ASF) under one or more contributor
```

Comment:
@lujiajing1126 The test cases could help you understand how to use the new `stream` module to query data. They show us three scenarios:

 * Query data by entities which are composited by `service_id`, `service_instance_id`, and `state`
 * Query data by local indices. For example, `duration` and `endpoint_id`. You can sort data by the local indices(`duration`) or the time.
 * Query data by the global index. The `trace_id` is the only case for skywalking trace.